python/lsst/source/injection/inject_engine.py

```diff
@@ -503,6 +509,15 @@ def inject_galsim_objects_into_exposure(
                 )
             injection_draw_size = draw_size_max
         draw_sizes[i] = injection_draw_size
+        if injection_core_size > injection_draw_size:
+            if logger:
+                logger.debug(
+                    "Clipping core draw size for object at %s from %d to %d pixels.",
```

Comment:
Should this be just "Clipping draw size for object..."?

Reply:
If we're reserving "draw size" for galsim.bounds objects, maybe it should read "Clipping core size..." ?

Reply:
That's what I suggested above too (you may have seen my initial reply with a typo, but I edited it).